ui/src/views/Status.vue

```diff
@@ -46,88 +46,111 @@
       </cv-column>
     </cv-row>
     <cv-row>
-      <cv-column :md="4" :max="4">
-        <NsInfoCard
-          light
-          :title="$t('status.sftpgo_admin_url')"
-          :icon="Settings32"
-          :loading="loading.getConfiguration"
-          class="min-height-card"
-        >
-        <template slot="content">
-          <div class="card-rows">
-            <div class="card-row">
-            {{ $t("status.default_credentials") }}
-              <cv-tooltip
-                alignment="start"
-                direction="top"
-                :tip="$t('status.sftpgo_admin_login_tips')"
-                class="info tooltip-mg-left"
-              >
-              </cv-tooltip>
-            </div>
-            <div class="card-row">
-              <NsButton kind="ghost" :icon="Launch20" @click="goToSftpgoAdmin">
-                {{ $t("status.open_web_admin_page") }}
-              </NsButton>
+      <template  v-if="!sftpgo_service && ! loading.getConfiguration">
+        <cv-column :md="4" :max="4">
+          <NsInfoCard
+            light
+            :title="$t('status.sftpgo_not_running')"
+            :icon="Settings32"
+            :loading="loading.getConfiguration"
+            class="min-height-card"
+          >
+          <template slot="content">
+            <div class="card-rows">
+              <div class="card-row">
+                <NsButton kind="ghost" :icon="Launch20" @click="goToAppPage(instanceName, 'settings')">
```

Comment:
```suggestion
                <NsButton kind="ghost" :icon="ArrowRight20" @click="goToAppPage(instanceName, 'settings')">
```
We usually use `Launch20` icon only for external links